<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicSet</class>
 <widget class="QDialog" name="BasicSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 255);</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/cs351set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>51</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>BOSCH Rexroth CS351设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>221</x>
       <y>111</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>IP：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>169</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>端口 A：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>227</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>端口 B：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cs351Ip">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>114</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PortA">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>173</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PortB">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>231</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/RFID set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>174</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>端口号 ：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>56</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>RFID设置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RfidPort">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>178</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RfidIp">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>119</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>116</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>IP：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/serve set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>166</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>曲线服务器：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DataServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>111</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>108</y>
       <width>106</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>数据服务器：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>48</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>服务器设置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CurveServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>170</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AndonServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>230</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>226</y>
       <width>123</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>Andon服务器：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/barcode_set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BarcodeGun">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>127</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>   条码枪设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>124</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>有线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>无线</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_37">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/cha.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_38">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>295</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/67.bmp);
font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>pushButton_38</zorder>
  <zorder>pushButton_37</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
